### PR TITLE
[test-only] share items with share permission if resharing is disabled

### PR DESCRIPTION
**Pull request by ScharfViktor**
coverage for https://github.com/owncloud/enterprise/issues/6423

### CODE REVIEW

**Comment by ScharfViktor** [comment]
@butonic I expect 400 error as agreed. About error message: please let me know message text when you solve issue

File: tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature [hunk 1]
```diff
@@ -165,3 +165,50 @@ Feature: Share a file or folder that is inside a space
     When user "Alice" expires the last share
     Then the HTTP status code should be "200"
     And as "Brian" folder "Shares/folder" should not exist
+
+  @issue-enterprise-6423 @env-config
+  Scenario Outline: user cannot share items in the project space with share permission if resharing is denied
+    Given the config "OCIS_ENABLE_RESHARING" has been set to "false"
+    And user "Alice" has shared a space "share sub-item" with settings:
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature:172]: let's use noun for space names
```suggestion
    And user "Alice" has shared a space "project 101" with settings:
```

Comment (ScharfViktor) [reply tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature:172]: I haven't space `project 101`. In the Background section Alice creates `share sub-item` space which I use

Comment (saw-jan) [reply tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature:172]: we can do it later. 👍

**Review by saw-jan — APPROVED** [review]

**Comment by ScharfViktor** [comment]
now after merging #8412 test should be green

**Pull request merged**